README.md

```diff
@@ -39,48 +39,84 @@ In particular, please read the [Pull Request Guidelines](https://github.com/exer
 
 ## Exercism Go style
 
-Let's walk through the `leap` exercise to see what is included in an exercise
-implementation. Navigate into the *leap* directory, you'll see there are a
-number of files there.
+Let's walk through the an imaginary `exercise-slug` exercise to see what is included in an exercise
+implementation. In any exercise you may see a number of files present:
 
 ```sh
-~/exercism/go/leap
+~/exercism/go/exercise-slug
 $ tree -a
 .
 ├── cases_test.go
 ├── example.go
-├── leap.go
-├── leap_test.go
-└── .meta
-    └── gen.go
-
-1 directory, 5 files
+├── exercise_slug.go
+├── exercise_slug_test.go
+├── .meta
+│   └── description.md
+│   └── gen.go
+│   └── hints.md
+│   └── metadata.yml
+└── README.md
 ```
 
-This list of files can vary across exercises so let's quickly run through
-each file and briefly describe what it is.
-
-* **cases_test.go** - This file contains [generated test cases](#generating-test-cases),
-    and will only be present in some exercises.
-
-* **example.go** - This is a reference solution for the exercise. This file is
-    ignored by the `exercism fetch` command. See [ignored files](#ignored-files) for
-    details on which files are ignored.
+This list of files *can vary* across exercises. Not all exercise
+utilize all of the available files, but they are available for use
+should an exercise need to adjust itself against [the global problem-specification repository](https://github.com/exercism/problem-specifications/tree/master/exercises). This
+repository helps collect common information for all exercises across
+all tracks. However, should local track-specific modifications or
+enhancements be made with respect to the global problem specifications
+they may be performed by the presence of specifically named files in
+an exercises `.meta/` directory.
```

Comment:
These paragraphs are a little unclear.


Not all exercises use all of these files. Exercises originate their test data from the Exercism problem-specification repository. This repository collects common information for all exercises across all tracks. 
However, should track-specific documentation need to be included with the exercise, files in an exercise's `.meta/` directory can be used.